test/e2e/lexicon-new-project.spec.ts

```diff
@@ -0,0 +1,537 @@
+      await expect(newLexProjectPageMember.srCredentialsPage.projectSelect).not.toBeVisible();
+
+      // Can go back to Chooser page
+      await expect(newLexProjectPageMember.backButton).toBeVisible();
+      await newLexProjectPageMember.backButton.click();
+      await expect(newLexProjectPageMember.chooserPage.sendReceiveButton).toBeVisible();
+    });
+  });
+
+  // step 1: send receive credentials
+  test.describe('Send Receive Credentials page', () => {
+
+    test.beforeEach(async () => {
+      await newLexProjectPageMember.goto();
+      await newLexProjectPageMember.chooserPage.sendReceiveButton.click();
+    });
+
+    test('Cannot move on if Password is empty', async () => {
+      await newLexProjectPageMember.expectFormStatusHasNoError();
+      await expect(newLexProjectPageMember.nextButton).toBeEnabled();
+      await newLexProjectPageMember.nextButton.click();
+
+      await expect(newLexProjectPageMember.srCredentialsPage.loginInput).toBeVisible();
+      await expect(newLexProjectPageMember.srCredentialsPage.projectSelect).not.toBeVisible();
+      await newLexProjectPageMember.expectFormStatusHasError();
+      await expect(newLexProjectPageMember.formStatus).toContainText('Password cannot be empty.');
+    });
+
+    test('Cannot move on if username is incorrect and can go back to Chooser page, user and password preserved', async ({ member }) => {
+      // passwordInvalid is, incredibly, an invalid password.
+      // It's valid only in the sense that it follows the password rules
+      await newLexProjectPageMember.srCredentialsPage.passwordInput.type(constants.passwordValid);
+      // tab should trigger validation of the password thus making the test less flaky
+      await newLexProjectPageMember.srCredentialsPage.passwordInput.press('Tab');
+      await expect(newLexProjectPageMember.srCredentialsPage.credentialsInvalid).toBeVisible();
+      await newLexProjectPageMember.expectFormStatusHasNoError();
+      await newLexProjectPageMember.nextButton.click();
+      await expect(newLexProjectPageMember.srCredentialsPage.loginInput).toBeVisible();
+      await expect(newLexProjectPageMember.srCredentialsPage.projectSelect).not.toBeVisible();
+      await newLexProjectPageMember.expectFormStatusHasError();
+      await expect(newLexProjectPageMember.formStatus).toContainText('The username or password isn\'t valid on LanguageDepot.org.');
+
+      // can go back to Chooser page, user and password preserved
+      await expect(newLexProjectPageMember.backButton).toBeVisible();
+      await newLexProjectPageMember.backButton.click();
+      await expect(newLexProjectPageMember.chooserPage.sendReceiveButton).toBeVisible();
+      await newLexProjectPageMember.chooserPage.sendReceiveButton.click();
+      await expect(newLexProjectPageMember.srCredentialsPage.loginInput).toBeVisible();
+      expect(await newLexProjectPageMember.srCredentialsPage.loginInput.inputValue()).toEqual(member.username);
```

Comment:
```suggestion
      await expect(newLexProjectPageMember.srCredentialsPage.loginInput).toHaveValue(member.username);
```
I think this is how you do this without losing playwright's "auto retry"

Reply:
Yes, you're right! Thank you. Fixed this in all test files in [this commit](https://github.com/sillsdev/web-languageforge/pull/1456/commits/39dbf4067525981b21070783cb9302dd03149caa).